README.md

```diff
@@ -49,7 +49,7 @@ The [reMarkable](https://www.remarkable.com) is a paper tablet for those who pre
 - [reMarkable-assistant](https://github.com/richeymichael/remarkable-assistant) - Manage templates, splash screens, and settings on your reMarkable tablet.
 - [reMarkable_entware](http://github.com/evidlo/remarkable_entware) - Package manager for reMarkable.  Install common Unix utilities through the command line.
 - [reMarkable-fs](https://github.com/nick8325/remarkable-fs) - A FUSE filesystem wrapper for the reMarkable tablet.
-- [reMarkable-hyutilities](https://github.com/moovida/remarkable-hyutilities) - A GUI written in java to backup your device, upload templates and modify spash screens.
+- [reMarkable-hyutilities](https://github.com/moovida/remarkable-hyutilities) - A GUI written in java to backup your device, upload templates and modify splash screens.
```

Comment:
While you're at it, consider also these changes:

```suggestion
- [reMarkable-hyutilities](https://github.com/moovida/remarkable-hyutilities) - A GUI written in Java to back up your device, upload templates and modify splash screens.
```

([reference](https://en.wiktionary.org/wiki/backup#Verb) for "backup" vs "back up")